golos.ctrl/src/golos.ctrl.cpp

```diff
@@ -221,6 +221,7 @@ void control::update_auths() {
         if (is_active) {
             auth.accounts.push_back({{_self, config::code_name}, thrs});    // add eosio.code
             auth.accounts.push_back({{config::emission_name, config::code_name}, thrs});    // needed to create tokens
+            auth.accounts.push_back({{config::publication_name, config::code_name}, thrs});    // needed to pay rewards
```

Comment:
publication contract is not constant, it's name depends on `_token` and placed into `_owner` field. as I remember, it must have `eosio.code` already set

Reply:
gls.publish must add gls.publish@eosio.code into active authority for pay rewards.